tests/drivers/build_all/pwm/prj.conf

```diff
@@ -1,3 +1 @@
-CONFIG_TEST=y
```

Comment:
This should be left in. `CONFIG_TEST` enables all sorts of asserts.

Reply:
then we need to remove some drivers/platforms, because the resulting binary does not fit.

Reply:
btw, this is  build only test, that does not do anything useful, it just builds the drivers, add CONFIG_TEST, add a whole lot of other things into the mix and it fails to link on some boards..

Reply:
Okay, fair points.